contracts/colonyNetwork/ColonyNetwork.sol

```diff
@@ -453,36 +435,33 @@ contract ColonyNetwork is BasicMetaTransaction, ColonyNetworkStorage, Multicall
     }
 
     if (isMiningChain()) {
-      uint128 nParents = skills[_skillId].nParents;
-      // We only update child skill reputation if the update is negative, otherwise just set nChildren to 0 to save gas
-      uint128 nChildren = _amount < 0 ? skills[_skillId].nChildren : 0;
-
-      IReputationMiningCycle(inactiveReputationMiningCycle).appendReputationUpdateLog(
-        _user,
-        _amount,
-        _skillId,
-        msgSender(),
-        nParents,
-        nChildren
-      );
+      appendReputationUpdateLogInternal(_user, _amount, _skillId, msgSender());
     } else {
       // Send transaction to bridge.
       // Call appendReputationUpdateLogFromBridge on metacolony on xdai
       // TODO: Maybe force to be set on deployment?
       require(miningBridgeAddress != address(0x0), "colony-network-foreign-bridge-not-set");
 
       reputationUpdateCount[getChainId()][msgSender()] += 1;
-      // require(bridgeData[bridgeAddress].chainId == MINING_CHAIN_ID, "colony-network-foreign-bridge-not-set-correctly");
       // Build the transaction we're going to send to the bridge
       bytes memory payload = abi.encodePacked(
         bridgeData[miningBridgeAddress].updateLogBefore,
-        abi.encodeWithSignature("appendReputationUpdateLogFromBridge(address,address,int256,uint256,uint256)", msgSender(), _user, _amount, _skillId, reputationUpdateCount[getChainId()][msgSender()]),
+        abi.encodeWithSignature(
+          "appendReputationUpdateLogFromBridge(address,address,int256,uint256,uint256)",
+          msgSender(),
+          _user,
+          _amount,
+          _skillId,
+          reputationUpdateCount[getChainId()][msgSender()]
+        ),
         bridgeData[miningBridgeAddress].updateLogAfter
       );
+
       (bool success, ) = miningBridgeAddress.call(payload);
       if (!success || !isContract(miningBridgeAddress)) {
         // Store to resend later
-        pendingReputationUpdates[getChainId()][msgSender()][reputationUpdateCount[getChainId()][msgSender()]] = PendingReputationUpdate(_user, _amount, _skillId, msgSender(), block.timestamp);
+        PendingReputationUpdate memory pendingReputationUpdate = PendingReputationUpdate(_user, _amount, _skillId, msgSender(), block.timestamp);
+        pendingReputationUpdates[getChainId()][msgSender()][reputationUpdateCount[getChainId()][msgSender()]] = pendingReputationUpdate;
```

Comment:
Interested in the gas cost here. The intention was to directly allocate to storage, but if under-the-hood that uses memory as well, then I agree this is better to read.

Reply:
I am not sure the difference, I was aiming for readability.

Reply:
40 gas more expensive to do it via memory.

Reply:
Ok, I'd vote to keep the more readable implementation then.